language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/contexts/RenameContextImpl.java

```diff
@@ -45,8 +45,9 @@
         super(operation, fileUri, wsManager, cursorPos, serverContext);
         this.clientCapabilities = clientCapabilities;
         this.params = params;
-        this.honorsChangeAnnotations = clientCapabilities.getTextDocCapabilities().getRename() != null &&
-                clientCapabilities.getTextDocCapabilities().getRename().getHonorsChangeAnnotations();
+        this.honorsChangeAnnotations = clientCapabilities.getTextDocCapabilities().getRename() != null
```

Comment:
What if we keep `honorsChangeAnnotations` in `boolean` primitive type instead of `Boolean` as well?